app/views/games/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<br>
+<br>
+
+<div class="text-center players">
+  <h2>&#9819; Player1 vs &#9813; Player2</h2>
```

Comment:
as devise is not yet marged - we dont have the user model set, so I just wrote Player 1 & 2 and temporal placeholders